numpy/core/tests/test_ufunc.py

```diff
@@ -1283,6 +1283,10 @@ def test_reduce_noncontig_output(self):
         assert_equal(y_base[1,:], y_base_copy[1,:])
         assert_equal(y_base[3,:], y_base_copy[3,:])
 
+    def test_no_doc_string(self):
+        # https://github.com/numpy/numpy/pull/9337
```

Comment:
Right, I wondered what gh- referred to in the other tests, obvious in retrospect.